src/sparse/node.rs

```diff
@@ -159,6 +204,20 @@ impl Node {
         &self.buffer
     }
 
+    fn bytes_height_mut(&mut self) -> &mut [u8] {
+        let range = Self::height_range();
+        &mut self.buffer_mut()[range]
+    }
+
+    fn bytes_height(&self) -> &[u8] {
+        let range = Self::height_range();
+        &self.buffer()[range]
+    }
+
+    fn set_bytes_height(&mut self, bytes: &Bytes4) {
+        self.bytes_height_mut().clone_from_slice(bytes)
+    }
+
```

Comment:
nit: These are not used for anything else than non-`bytes_` versions of the calls and don't see any usecase where these would be needed separately. Maybe consider merging them together and removing the `bytes_` versions?

Reply:
I get what you're saying. I implemented it this way to maintain the pattern I use for the other byte fields within the buffer, and this keeps it symmetric. I can consider refactoring this pattern in the future to see if it simplifies the impl as a whole, but for this PR I think I would like to maintain the pattern as is.